lib/sqlalchemy/dialects/postgresql/json.py

```diff
@@ -340,4 +364,28 @@ def contained_by(self, other):
                 CONTAINED_BY, other, result_type=sqltypes.Boolean
             )
 
+        def delete_path(self, array):
+            """JSONB expression. Deletes field or array element specified in
+            the argument array.
+            """
+            return self.operate(DELETE_PATH, array, result_type=JSONB)
+
+        def path_exists(self, other):
+            """Boolean expression. Test for presence of item given by the
+            argument JSONPath expression.
+            """
+            return self.operate(
+                PATH_EXISTS, other, result_type=sqltypes.Boolean
+            )
```

Comment:
**mike bayer** (zzzeek) wrote:

i was sort of hoping we had a more central place to do a coercion like this but it seems that we don't for a dialect-specific thing like this, will see how this goes.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4336